src/res_tailwindcss.ml

```diff
@@ -64,7 +64,7 @@ let expand ~ctxt label =
   if String.length stripped_label = 0 then
     Ast_builder.Default.estring ~loc label
   else
-    let classnames = stripped_label |> String.split ~on:' ' in
+    let classnames = stripped_label |> Str.global_replace (Str.regexp "[ \n\r\x0c\t]+") " " |> String.split ~on:' ' in
```

Comment:
```suggestion
    let classnames = stripped_label |> String.split ~on:' ' |> List.map ~f:(fun s -> String.strip x) in
```

I suggest this is more simple way and readable. What do you think?

Reply:
If strip and trim work the same, in this case "" className is needed. That's why it throws a compile error. If we do not want to use regexp, I think we can filter it and process it.

If we add the following className to tailwind.css it will compile.

`tailwind.css`
```css
. {
}
```

Reply:
comment test

Reply:
```ocaml
    let classnames = stripped_label |> String.split ~on:' ' |> List.map ~f:(fun x -> String.strip x) |> List.filter ~f:(fun x -> String.length x > 0) in
```

Reply:
Good point! How about this? We can use `Str.split` for this case.

```ocaml
    let classnames = stripped_label |> Str.split (Str.regexp "[ \t]+") in
```

And I think `"[ \t]+"` would be fair enough, others like `\n`, `\r`, `\x0c` should be failed to be compiled. 😆

Reply:
FYR,

```ocaml
val split : regexp -> string -> string list
(** [split r s] splits [s] into substrings, taking as delimiters
   the substrings that match [r], and returns the list of substrings.
   For instance, [split (regexp "[ \t]+") s] splits [s] into
   blank-separated words.  An occurrence of the delimiter at the
   beginning or at the end of the string is ignored. *)

```

Reply:
```ocaml
# "" |> Str.split (Str.regexp "[ \t]+") ;;
- : string list = []
#  "" |> Str.split (Str.regexp "[ \t]+") ;;
- : string list = []
#  "a b c  d" |> Str.split (Str.regexp "[ \t]+") ;;
- : string list = ["a"; "b"; "c"; "d"]
#  "a b c  d     f" |> Str.split (Str.regexp "[ \t]+") ;;
- : string list = ["a"; "b"; "c"; "d"; "f"]
# 
```
Thanks 😊